system/src/system_control_internal.cpp

```diff
@@ -152,6 +157,81 @@ int decodeRequestMessage(ctrl_request* req, const pb_field_t* fields, void* dst)
     return ret;
 }
 
+// Helper classes for working with nanopb's string fields
```

Comment:
Some more comments about what this does would be helpful.

Reply:
👍